mage_ai/cli/utils.py

```diff
@@ -33,3 +34,118 @@ def get_value(value: Union[str, None]) -> Any:
         except json.decoder.JSONDecodeError:
             pass
     return value
+
+
+def load_json(json_file: str) -> dict:
+    """
+    Load JSON file.
+
+    Args:
+        json_file (str): The JSON file path.
+
+    Returns:
+        json_data (dict): JSON data as a python dictionary.
+    """
+    with open(json_file, "r") as f:
+        json_data = json.load(f)
+
+    return json_data
+
+
+def generate_models_sql(catalog_data: dict, schema_name: str) -> list[tuple[str, str]]:
```

Comment:
It's actually better put these methods in this folder: https://github.com/mage-ai/mage-ai/tree/master/mage_ai/data_preparation/models/block/dbt/utils